kv/telegraf.go

```diff
@@ -195,6 +207,25 @@ func (s *Service) putTelegrafConfig(ctx context.Context, tx Tx, tc *influxdb.Tel
 	if err := bucket.Put(encodedID, v); err != nil {
 		return UnavailableTelegrafServiceError(err)
 	}
+
+	return s.putTelegrafConfigStats(encodedID, tx, tc)
+}
+
+func (s *Service) putTelegrafConfigStats(encodedID []byte, tx Tx, tc *influxdb.TelegrafConfig) error {
```

Comment:
Add tests for this and deleteTelegrafConfigStats

Reply:
worht giving hte new `StoreBase` or `IndexStore` a run for the money here:
base: https://github.com/influxdata/influxdb/blob/master/kv/store_base.go
index store: https://github.com/influxdata/influxdb/blob/master/kv/store_index.go
example use: https://github.com/influxdata/influxdb/blob/master/kv/check.go